src/main/java/net/sf/jabref/model/entry/InternalBibtexFields.java

```diff
@@ -305,17 +273,29 @@ private InternalBibtexFields(boolean serializeSpecialFields) {
             add(field);
         }
 
-        // collect all public fields for the PUBLIC_FIELDS array
-        for (BibtexSingleField sField : fieldSet.values()) {
-            if (!sField.isPrivate()) {
-                PUBLIC_FIELDS.add(sField.getFieldName());
-                // or export the complete BibtexSingleField ?
-                // BibtexSingleField.toString() { return fieldname ; }
-            }
-        }
+    }
 
-        // sort the entries
-        Collections.sort(PUBLIC_FIELDS);
+
+    public static void updateTimeStampField(String timeStampFieldName) {
```

Comment:
Yes you are right, your PR goes into the right direction. I just saw a new "static" method and thought that it was a bad idea (in the end it is, but the best possible with the current design). Sorry for that.